app/src/main/java/org/simple/clinic/patient/PatientAddress.kt

```diff
@@ -98,5 +98,8 @@ data class PatientAddress(
 
     @Query("SELECT COUNT(uuid) FROM PatientAddress")
     abstract fun count(): Int
+
+    @Query("SELECT DISTINCT colonyOrVillage FROM PatientAddress")
```

Comment:
Do we need to order them in any specific way? I'd assume alphabetical ordering is preferred, otherwise it would be hard to find a location from a long list of names.